java-tools/OsmAndServer/src/main/java/net/osmand/server/controllers/user/MapApiController.java

```diff
@@ -365,6 +365,7 @@ public ResponseEntity<String> listFiles(@RequestParam(required = false) String n
 		}
 		UserFilesResults res = userdataService.generateFiles(dev.userid, name, allVersions, true, type);
 		List <UserFileNoData> filesToIgnore = new ArrayList<>();
+		int cloudReads = 0, cacheWrites = 0;
```

Comment:
this logs is ok for branch testing but too much to write to main log probably

Reply:
Detailed logs are required to debug the web-list-files on production load.